app/config/colors.js

```diff
@@ -1,3 +1,5 @@
 export default {
   blue: '#0000FF',
+  black: '#000000',
+  green: '#00A778',
 }
```

Comment:
I had suggested all the colors used in the app, I'd look at my previous comment and just add all the needed colors in here now.

Reply:
as we are working will add colors according to need.

Reply:
I'd rather base colours  for the project are defined early on and add more as you go as needed. Your approach is more likely to lead to duplicates from other contributors.

Reply:
@Bhuvneshwar-MB are you going to add colors? If not il merge.

Reply:
Yes Michael. i am adding the colors.

Reply:
Can you be consistent in the naming and not have `Color` at the end? ie `separatorLineGrayColor` -> `separatorLineGray`

Reply:
Done